eng/Build.ps1

```diff
@@ -99,6 +99,7 @@ function Process-Arguments() {
        Print-Usage
        exit 0
     }
+    $script:nodeReuse = $False;
```

Comment:
I don't see where this is consumed.

Reply:
There's `nodeReuse` in `eng\common\msbuild.ps1` - what's the right way to get this value down to there?

Reply:
Also `tools.ps1`

Reply:
```
# Set to true to reuse msbuild nodes. Recommended to not reuse on CI.
[bool]$nodeReuse = if (Test-Path variable:nodeReuse) { $nodeReuse } else { !$ci }
```

Reply:
Note that on WIndows this doesn't strictly seem necessary - on Mac and Linux there are definitely problems with reusing nodes.